docs/websites.mdx

```diff
@@ -0,0 +1,133 @@
+---
+description: 'Hosting and managing websites with Nitric'
+---
+
+# Websites
+
+Nitric provides a simple way to deploy websites to the cloud, allowing you to serve website assets in a serverless environment without managing complex infrastructure.
+
+If you're interested in the architecture, provisioning, or deployment steps, they can be found [here](/architecture/websites).
+
+<Note>
+  At this time, websites can only be deployed as static assets. If you require
+  more advanced features, let us know:
+  https://github.com/nitrictech/nitric/issues
+</Note>
+
+## Overview
+
+Nitric’s website resource lets you:
+
+- Serve static files (HTML, JS, CSS, images) from cloud storage.
+- Service your entire nitric application from a single domain.
+- APIs are automatically served under `/api` and websites are served under `/`.
```

Comment:
A sub-heading in these docs would be useful, explaining how the API routes work, why they're useful (e.g. CORS).

This feature should probably also be optional and configurable (i.e. the /api route should be configurable incase it overlaps with an /api route in the static site).

Reply:
Yep lets create an issue for the configurable rewrite path